app/views/users/liked.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="row mb-4">
+  <div class="col-md-6 offset-md-3">
+    <%= render "users/user", user: @user %>
+  </div>
+</div>
+
+<div class="row mb-2">
+  <div class="col-md-6 offset-md-3">
+    <%= render "users/profile_nav", user: @user %>
+  </div>
+</div>
+
+<% @user.liked_photos.each do |photo| %>
```

Comment:
No if-statements to check if we follow the user or if their profile is public/private? I looked into the "photos/_photo" file, no checks there as well, wondering how we are hiding profiles of private users from non-followers?

Reply:
That is correct; if you look at the output of this code, you will see that there's no functional difference between public and private accounts. Everyone can see everything. In this codebase there isn't yet any authorization/permissioning going on.

This codebase is the _starting point_ for the industrial-auth-1 project, where you need to add those checks to match [the next target].

Reply:
Ahh yes true, thank you for that